index.js

```diff
@@ -9,11 +9,15 @@ var path = require('path')
 var inquirer = require('inquirer')
 var wordwrap = require('wordwrap')
 var assert = require('assertf')
+var fs = require('fs');
 
 module.exports = function(options) {
   // Prevent automatic close of stdin, so that input is possible after showing the problem description.
-  options.autoclose = false
+  //options.autoclose = false
```

Comment:
I forget exactly. Probably to allow users to interact with Inquirer prompt. I'll look into eventually but feel free to close this pr if it needs to much work as this is below "actually finish my workshop" on my priority list and that's below done other things so this probably won't receive attention for a while.